tests/kuttl/tests/ovn_restart_flow/04-assert-flows-present.yaml

```diff
@@ -0,0 +1,16 @@
+#
+# Check for:
+#
+# - Check that the flow added to the test bridge "br-test-flows" is present
+#   after the ovn-controller has been restarted.
+#   Expected flow: "table=100, priority=200 actions=drop"
+
+apiVersion: kuttl.dev/v1beta1
+kind: TestAssert
+timeout: 300
+commands:
+    - script: |
+        node=$(oc get nodes -o name|sort|head -1| sed "s|node/||g")
+        controller_pod=$(oc get pod -n $NAMESPACE -l service=ovn-controller-ovs --field-selector spec.nodeName=$node -o name | head -1)
+        expected_flows="table=100, priority=200 actions=drop"
+        oc rsh -n $NAMESPACE --container ovs-vswitchd $controller_pod ovs-ofctl dump-flows br-test-flows --no-stats | grep -q "$expected_flows" || exit 1
```

Comment:
let's also delete the test bridge after test

Reply:
Let me push a new PS

Reply:
I've pused a new PS. I was thinking about adding a trap in this script, but that means always deleting the bridge if the script exits. We only want to remove the bridge if the script first reads the flows correctly.

Reply:
> We only want to remove the bridge if the script first reads the flows correctly.

Why?

Re: trap: you can set trap and then remove it with `trap - EXIT`.